odoo/addons/base/ir/ir_binding.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from collections import defaultdict
+
+from odoo import api, fields, models
+from odoo.exceptions import AccessError, MissingError
+
+
+BINDING_TYPES = [
+    ('client_action_multi', "Sidebar action"),
+    ('client_print_multi', "Sidebar report print"),
+    ('client_action_relate', "Sidebar related link"),
```

Comment:
Change types to `action` and `report`; remove `client_action_relate`; make `action` the default.